src/CoursePageResources/index.scss

```diff
@@ -0,0 +1,48 @@
+.container-fluid {
+    background-color: #EAF3FB;
+    color: #2E8ED3;
```

Comment:
Colours shouldn't be hard-coded. Get them from the theme, or at least set them globally in a theme file.